logging-operator/templates/crd.yaml

```diff
@@ -1,18 +1,112 @@
 apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Please create separate crds. eg.: fluentbit-crd.yaml and fluentd-crd.yaml, plugins-crd.yaml